test-cases/performance/perf-regression-nosqlbench-throughput-mixed.yaml

```diff
@@ -0,0 +1,19 @@
+test_duration: 360
+
+stress_cmd_m: "nosqlbench cql-tabular2 rampup-cycles=1M main-cycles=100M write_ratio=5 read_ratio=5 async=300 --progress console:360s rf=3 partsize=5000 driver=cql-scylla -v"
+
+n_db_nodes: 3
+n_loaders: 1
```

Comment:
They did not specify the number of loaders. We might need to go up to 2 with this, but I'd start with 1. Once we have the e-mail and ES exports in place, we'll be able to compare our numbers to the ones in the benchmark article and adjust.

Reply:
We don't need to wait for the email and ES export for that.
We should run it with live monitor in a loop, of running the load and adding more loaders.
When adding the loaders doesn't increase the throughput, this is the peak throughput of the test.

Reply:
So I think we have 2 cases here: one is simply mimicking the k8ssandra tests and the other is doing a separate throughput test, trying to saturate the cluster. As for the k8ssandra test, when running with 1 loader I saw `docker stats` clocking the cpu usage at 500-600% (which means we were using up 5/6 out of the 8 vCPUs on the loader). My suggestion would be: let's leave this configuration and rename it as something like `perf-regression-mixed-50r-50w-k8ssandra-reference.yaml` test case, and have another that would aim at saturating the cluster with more loaders. What do you think?

Reply:
So, you need to check the result with one loader, if the cluster isn't saturated, you need to add another loader (regardless of their blog, we don't need two variants of the test, because a throughput test is worthless if you don't saturate the throughput).

If the loader is saturated, you should try with a stronger loader instance type as well.

Reply:
I added a second loader. The cluster load under test is at around 70% now. I'm tweaking the nosqlbench command to push for more throughput.